Gems/ROS2/Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -0,0 +1,485 @@
+                &AzToolsFramework::AssetSystem::AssetSystemRequest::GetSourceInfoBySourcePath,
+                azMeshPath.c_str(),
+                assetInfo,
+                watchDir);
+
+            // Set export method to convex mesh
+            auto readOutcome = AZ::JsonSerializationUtils::ReadJsonFile(assetInfoFilePath.c_str());
+            if (!readOutcome.IsSuccess())
+            {
+                AZ_Error(
+                    Internal::collidersMakerLoggingTag,
+                    false,
+                    "Could not read %s with %s",
+                    assetInfoFilePath.c_str(),
+                    readOutcome.GetError().c_str());
+                return;
+            }
+            rapidjson::Document assetInfoJson = readOutcome.TakeValue();
+            auto manifestObject = assetInfoJson.GetObject();
+            auto valuesIterator = manifestObject.FindMember("values");
+            if (valuesIterator == manifestObject.MemberEnd())
+            {
+                AZ_Error(
+                    Internal::collidersMakerLoggingTag, false, "Invalid json file: %s (Missing 'values' node)", assetInfoFilePath.c_str());
+                return;
+            }
+
+            constexpr AZStd::string_view physXMeshGroupType = "{5B03C8E6-8CEE-4DA0-A7FA-CD88689DD45B} MeshGroup";
+            auto valuesArray = valuesIterator->value.GetArray();
+            for (auto& value : valuesArray)
+            {
+                auto object = value.GetObject();
+
+                auto physXMeshGroupIterator = object.FindMember("$type");
+                if (AZ::StringFunc::Equal(physXMeshGroupIterator->value.GetString(), physXMeshGroupType))
+                {
+                    value.AddMember(rapidjson::StringRef("export method"), rapidjson::StringRef("1"), assetInfoJson.GetAllocator());
+                }
+            }
+
+            auto saveOutcome = AZ::JsonSerializationUtils::WriteJsonFile(assetInfoJson, assetInfoFilePath.c_str());
+            if (!saveOutcome.IsSuccess())
+            {
+                AZ_Error(
+                    Internal::collidersMakerLoggingTag,
+                    false,
+                    "Could not save %s with %s",
+                    assetInfoFilePath.c_str(),
+                    saveOutcome.GetError().c_str());
+                return;
+            }
+
+            // Add asset to expected assets list
+            if (assetFound)
+            {
+                AZStd::lock_guard lock{ m_buildMutex };
+                m_meshesToBuild.push_back(AZ::IO::Path(assetInfo.m_relativePath));
+            }
+        }
+    }
+
+    void CollidersMaker::AddColliders(urdf::LinkSharedPtr link, AZ::EntityId entityId)
+    {
+        AZStd::string typeString = "collider";
+        const bool isWheelEntity = Utils::IsWheelURDFHeuristics(link);
+        if (isWheelEntity)
+        {
+            AZ_Printf(Internal::collidersMakerLoggingTag, "Due to its name, %s is considered a wheel entity", link->name.c_str());
```

Comment:
repeat: `AZ_Printf` should have newline at the end of the format string.
```suggestion
            AZ_Printf(Internal::collidersMakerLoggingTag, "Due to its name, %s is considered a wheel entity\n", link->name.c_str());
```